rest_framework_simplejwt/locale/fa_IR/LC_MESSAGES/django.po

```diff
@@ -4,108 +4,112 @@ msgid ""
 msgstr ""
 "Project-Id-Version: djangorestframework_simplejwt\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-02-22 17:30+0100\n"
-"Last-Translator: Hirad Daneshvar <hirad.daneshvar@gmail.com>\n"
+"POT-Creation-Date: 2023-07-04 09:34+0330\n"
+"Last-Translator: Mahdi Rahimi <mahdi.rahimi95@gmail.com>\n"
 "Language: fa_IR\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: authentication.py:78
+#: authentication.py:89
 msgid "Authorization header must contain two space-delimited values"
 msgstr "هدر اعتبارسنجی باید شامل دو مقدار جدا شده با فاصله باشد"
 
-#: authentication.py:104
+#: authentication.py:115
 msgid "Given token not valid for any token type"
 msgstr "توکن داده شده برای هیچ نوع توکنی معتبر نمی‌باشد"
 
-#: authentication.py:116 authentication.py:143
+#: authentication.py:127 authentication.py:162
 msgid "Token contained no recognizable user identification"
 msgstr "توکن شامل هیچ شناسه قابل تشخیصی از کاربر نیست"
 
-#: authentication.py:121
+#: authentication.py:132
 msgid "User not found"
 msgstr "کاربر یافت نشد"
 
-#: authentication.py:124
+#: authentication.py:135
 msgid "User is inactive"
 msgstr "کاربر غیرفعال است"
 
-#: backends.py:67
+#: authentication.py:142
+msgid "The user's password has been changed."
+msgstr "رمز عبور کاربر تغییر کرده است"
+
+#: backends.py:69
 msgid "Unrecognized algorithm type '{}'"
 msgstr "نوع الگوریتم ناشناخته '{}'"
 
-#: backends.py:73
+#: backends.py:75
 msgid "You must have cryptography installed to use {}."
 msgstr "برای استفاده از {} باید رمزنگاری را نصب کرده باشید."
 
-#: backends.py:88
+#: backends.py:90
 msgid ""
 "Unrecognized type '{}', 'leeway' must be of type int, float or timedelta."
 msgstr ""
```

Comment:
missing translation

Reply:
This text does not give the correct meaning in Persian language.

Reply:
solve it.